templates/core/index.html

```diff
@@ -1,817 +1,754 @@
-          alt="",
-          width="71",
-          height="60",
-          hi_def=True,
-          loading="lazy"
-          ) | safe
-        }}
+      <div class="p-equal-height-row__item">
+        <hr class="p-rule--highlight u-hide--medium u-hide--small is-muted" />
+        <h3 class="p-heading--5">Agile containerisation</h3>
       </div>
-    </div>
-    <div class="col-3">
-      <div class="u-sv2">
-        {{ image (
-          url="https://assets.ubuntu.com/v1/e39199bc-Internet+of+Things_digital-gateways.svg",
-          alt="Digital gateways",
-          width="80",
-          height="70",
-          hi_def=True,
-          loading="lazy"
-          ) | safe
-          }}
+      <p class="p-equal-height-row__item">
+        Ubuntu Core is immutable and strictly confined. There is a clean separation between the kernel, OS image and
+        applications, each updated independently and protected against corruption.
+      </p>
+      <div class="p-equal-height-row__item">
+        <p>
+          <a href="/core/docs/components#heading--confinement">Learn more about confinement&nbsp;&rsaquo;</a>
+        </p>
       </div>
     </div>
-    <div class="col-3">
-      <div class="u-sv2">
-        {{ image (
-          url="https://assets.ubuntu.com/v1/38efd13b-solution-robot.svg",
-          alt="",
-          width="51",
-          height="81",
-          hi_def=True,
-          loading="lazy"
-          ) | safe
-        }}
+    <div class="p-equal-height-row__col">
+      <div class="p-equal-height-row__item u-hide--small u-hide--medium">
+        <div class="p-media-container">
+          <img width="320" src="https://assets.ubuntu.com/v1/166ec7c6-hardware-ecosystem.jpg" alt="" />
+        </div>
+      </div>
+      <div class="p-equal-height-row__item">
+        <hr class="p-rule--highlight u-hide--medium u-hide--small is-muted" />
+        <h3 class="p-heading--5">A strong hardware ecosystem</h3>
+      </div>
+      <p class="p-equal-height-row__item">
+        We enable Ubuntu Core with the best ODMs and silicon vendors in the world. We continuously test it on leading
+        IoT and edge devices and hardware.
+      </p>
+      <div class="p-equal-height-row__item">
+        <p>
+          <a href="/certified">Browse all certified hardware&nbsp;&rsaquo;</a> 
+        </p>
       </div>
     </div>
   </div>
-  <div class="row u-align--center">
-    <div class="col-3">
-      <p class="p-heading--4 u-sv2"><a href="/automotive">Automotive&nbsp;&rsaquo;</a></p>
-    </div>
-    <div class="col-3">
-      <p class="p-heading--4 u-sv2"><a href="/industrial">Industrial&nbsp;&rsaquo;</a></p>
-    </div>
-    <div class="col-3">
-      <p class="p-heading--4 u-sv2"><a href="/internet-of-things">IoT Gateways&nbsp;&rsaquo;</a></p>
-    </div>
-    <div class="col-3">
-      <p class="p-heading--4 u-sv2"><a href="robotics">Robotics&nbsp;&rsaquo;</a></p>
+  <hr class="p-rule is-muted" />
```

Comment:
```suggestion
<div class="u-fixed-width">
  <hr class="p-rule is-muted" />
</div>
```
Has to be wrapped like this since it's outside of a row. Otherwise it doesn't align to the grid correctly 
![image](https://github.com/canonical/ubuntu.com/assets/17607612/070a6441-7c3b-4ba3-bed9-ca01c9845553)